packages/eslint-plugin-khan/package.json

```diff
@@ -14,10 +14,17 @@
   "module": "dist/es/index.js",
   "author": "Kevin Barabash <kevinb@khanacademy.org>",
   "license": "MIT",
+  "dependencies": {
+    "ancesdir": "^5.0.1",
+    "@babel/types": "^7.23.0",
+    "@typescript-eslint/utils": "8.17.0",
+    "checksync": "^5.0.5"
+  },
```

Comment:
These were moved up from below `devDependencies`, but also added the `ancesdir` dep. It used to work with `yarn` but `pnpm` lays out the `node_modules` dir differently and so we no longer see another package's (or root's) dependencies and we need to be explicit.